### DISCUSSION TITLE
AJUDA - sou iniciante

### DISCUSSION THREAD

**Discussion by EduardoHFG** [apyb/comunidade#171] [Pergunta]
oi gente beleza eu estou começando a estudar python e gostaria de pedir ajuda para conseguir fazer um code simples e sabe como faz para colocar mais de ''significado'' a uma variável por exemplo nesse code q eu fiz : 

e caso tenha como eu melhorar ele me falem!! 

`n1 = int(input('digite um numero para somar com outro'))
n2 = int(input('digite seu outro numero'))
mm = input('escolha se vai ser mais ou menos a sua conta')

tm = n1+n2
tn = n1-n2

if mm == ('+','mais'):
    print(tm)

elif mm == ('-','menos') :
    print(tn)

else:
    print('voce digitou algo errado tente novamente') 
`

**Comment by cantoniazzi** [comment]
Fala, @EduardoHFG. Tudo bem?

O que você quer dizer com `colocar mais de ''significado'' a uma variável `?

Seria sobre verificar dois valores da variável `mm` aqui nesse seu condicional `if mm == ('+','mais'):` ?

Uma forma correta de verificar dois valores em um condicional seria assim:
```
if mm == '+' or mm == 'mais':
```
No if ali eu verifico se a variável tem o valor '+' ou se ela tem o outro valor 'mais'.

Uma forma de verificar também (e fazendo em uma única condição no if) seria assim:

```
if mm in ['+', 'mais']:
```
Que aí você está verificando se o valor da variável `mm` está contido na lista de valores ali onde coloquei `'+'` e `'mais'`.

Conseguiu compreender?  Pode ir mandando suas dúvidas aqui, que nós vamos tentando te ajudar. 🕺🏾
